Documentation/ApiOverview/GlobalValues/Typo3ConfVars/MAIL.rst

```diff
@@ -297,10 +299,19 @@ $GLOBALS['TYPO3_CONF_VARS']['MAIL']['dsn']
    :Default: ''
 
    *only with transport=dsn* The DSN configuration of the Symfony mailer
-   (eg. smtp//userpass@smtp.example.com25). For 3rd party transports you
-   have to add additional dependencies. See
-   `Symfony mailer <https//symfony.com/doc/current/mailer.html>`__ for more
-   details.
+   (for example smtp//userpass@smtp.example.com25). Symfony provides different
```

Comment:
are there colons missing in this config?

Reply:
yeah seems like I accidently removed them on converting from yaml to rst. added them again